scripts/logs/logs-errors.sh

```
@@ -0,0 +1,205 @@
+#!/bin/bash
+# Error-focused log filtering with context
+# Usage: ./logs-errors.sh [--context N] [--since TIME] [--help]
+
+# Colors
+RED='\033[0;31m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
```

Shellcheck (0.10.0)

[warning] BLUE appears unused. Verify use (or export if used externally).

(SC2034)